### ISSUE TITLE
Detect Version at runtime

### ISSUE THREAD

**Issue by JoergTiedemann** [volkszaehler/libsml#161]
Is ther any functionality to detect the current version of this library at runtime ?

**Comment by r00t-** [comment]
obviously not, else it would be here i guess:
https://github.com/volkszaehler/libsml/blob/master/sml/include/sml/sml_shared.h

adding a function would be an ABI change and prevent linking against older version...

i guess you want this to be able to check if some bugfix is present ot not?

**Comment by JoergTiedemann** [comment]
Yes need it just to check what’s running and if some bugging is present,
Would be great if such a function will be added
